components/os/windows.go

```diff
@@ -170,6 +179,34 @@ func (f *agentURLFinder) findVersion(fullVersion string) (string, error) {
 	return url, nil
 }
 
+func getAgentURLFromPipelineID(pipeline string) (string, error) {
+	// FIXME: remove pipeline- from the pipelineID we do not want it for Windows
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Can you open a JIRA card to fix this ?

Reply:
Sure will create it